website/addons/dropbox/utils.py

```diff
@@ -43,10 +43,14 @@ def log(self, action, extra=None, save=False):
         :param dict extra: Extra parameters to add to the ``params`` dict of the
             new NodeLog.
         """
+        if self.node.get_addon('dropbox', deleted=True).folder == '/':
+            folder = '/(Full DropBox)'
```

Comment:
I don't think the parenthetical should go in the log; it is only used for human-readable display. Furthermore, it would lead to inconsistency in how we store logs (some logs would have '/' for root; others would have '/Full DropBox').